ScnSoft-Education/LeetCodeTraining/ReverseInteger/ReverseIntegerSolution.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System.Text;
+
+namespace LeetCodeTraining.ReverseInteger
+{
+    //https://leetcode.com/problems/reverse-integer/
+    public static class ReverseIntegerSolution
+    {
+        public static int Reverse(int x)
+        {
+            string positiveString = x.ToString();
+            int sign = 1;
+            StringBuilder builder = new StringBuilder();
+
+            if (x < 0)
+            {
+                sign = -1;
+                positiveString = (-1 * x).ToString();
```

Comment:
it seems could be better option here as:
positiveString = positiveString .Substring(1);